.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.0
```

Comment:
Remove this file, we run tests with various versions.

Reply:
done